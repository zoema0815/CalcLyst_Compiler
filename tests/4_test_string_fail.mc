int main () {
    int x;
    String w;
    x = 3;
    w = x;
    return 0;
}