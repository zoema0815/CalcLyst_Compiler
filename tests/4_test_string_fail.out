Fatal error: exception Failure("illegal assignment String = int in w = x")
